src/frontend/src/components/plugins/PluginUIFeature.tsx

```diff
@@ -84,7 +84,7 @@ export const getPluginTemplateEditor = (
     useEffect(() => {
       (async () => {
         try {
-          await func({
```

Comment:
Isn't the await needed here so that asynchronous errors are catched too? But not sure about it.

Reply:
Shouldn't `func` be marked async for that?

Reply:
This can be asynchronous if 

https://github.com/inventree/InvenTree/blob/master/src/frontend/src/components/plugins/PluginUIFeatureTypes.ts#L1is defined as a promise by the individual feature.

Reply:
But you’re right, in this case it can’t be async, so its fine removing it. Good catch.

Reply:
TBH all of those were caught by SonarCloud and were marked as middle to high importance so I mainly fixed them to get the quality gate back to good-ish before the release